packages/app/src/entities/token/utils/get-color-by-token-name.tsx

```diff
@@ -0,0 +1,10 @@
+import { SupportedToken } from '@/shared/stellar/constants/tokens'
+
+export const getColorByTokenName = (tokenName: SupportedToken) => {
+  const colorByToken = {
```

Comment:
```suggestion
  const colorByToken: Record<SupportedTokens, `#${string}`> = {
```
Or you can write the return type of the function.

I'm not sure about the reason of making a function in such cases instead of objects with Record type.

Reply:
moved to constants

Reply:
oh, nice